src/darwin/Framework/CHIP/MTRBaseDevice_Internal.h

```diff
@@ -228,7 +238,7 @@ static inline MTRTransportType MTRMakeTransportType(chip::Transport::Type type)
 
 // Exported utility function
 // Convert TLV data into data-value dictionary as described in MTRDeviceResponseHandler
-NSDictionary<NSString *, id> * _Nullable MTRDecodeDataValueDictionaryFromCHIPTLV(chip::TLV::TLVReader * data);
+NSDictionary<NSString *, id> * _Nullable MTRDecodeDataValueDictionaryFromCHIPTLV(chip::TLV::TLVReader * data, NSNumber * _Nullable dataVersion = nil);
```

Comment:
It would be great if we could move the various utilities related to data-value dictionaries into their own header and implementation file (maybe a follow-up?)

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/32154